### DISCUSSION TITLE
How to capture GPT's action?

### DISCUSSION THREAD

**Discussion by growmuye** [LouisShark/chatgpt_system_prompt#25] [Q&A] [answered]
<img width="277" alt="image" src="https://github.com/LouisShark/chatgpt_system_prompt/assets/79145285/40a9c30e-6238-4771-8268-e8bceeed9db2">

**Comment by LouisShark** [comment, marked as answer]
[donanimhaber.com](https://github.com/LouisShark/chatgpt_system_prompt/graphs/traffic?referrer=donanimhaber.com#top-domains) @growmuye if you want to get action's schema, you can try

**Comment by 0xeb** [comment]
One easy way is to just right click / view source code (HTML) of the ChatGPT GPT; make sure you open an incognito window (or you are not signed in). You will then find a JSON blob there. If there are actions you will see their definitions.

![image](https://github.com/LouisShark/chatgpt_system_prompt/assets/230851/4bcf37cb-49f0-4ad0-94c5-5161c61d8bb9)

Closing this discussion too.
